metacat-elasticsearch/src/main/java/com/netflix/metacat/elasticsearch/search/ElasticSearchDoc.java

```diff
@@ -94,33 +92,8 @@ public ElasticSearchDoc(final String id, final Object dto, final String user, fi
         this.refreshMarker = refreshMarker;
     }
 
-    private static Class getClass(final String type) {
-        return Type.valueOf(type).getClazz();
-    }
-
-    /**
-     * Parse the elastic search response.
-     *
-     * @param response response
-     * @return document
-     */
-    public static ElasticSearchDoc parse(final GetResponse response) {
-        ElasticSearchDoc result = null;
-        if (response.isExists()) {
-            final Map<String, Object> responseMap = response.getSourceAsMap();
-            final String user = (String) responseMap.get(Field.USER);
-            final boolean deleted = (boolean) responseMap.get(Field.DELETED);
-            @SuppressWarnings("unchecked") final Object dto = MetacatJsonLocator.INSTANCE.parseJsonValue(
-                    response.getSourceAsBytes(),
-                    getClass(response.getType())
-            );
-            result = new ElasticSearchDoc(response.getId(), dto, user, deleted);
-        }
-        return result;
-    }
-
     private ObjectNode toJsonObject() {
```

Comment:
These methods seem like they should be Utility methods to me. Shouldn't put the JSON serialization logic inside the object. perhaps that's just my opinion though